_scripts/toolbox-create

```diff
@@ -8,7 +8,8 @@ run="toolbox run -c cockpit-website"
 echo "## Installing RPM dependencies inside container..."
 $run sudo dnf install -y rubygem-bundler \
 	ruby-devel libffi-devel make gcc gcc-c++ redhat-rpm-config zlib-devel \
-	libxml2-devel libxslt-devel tar nodejs
+	libxml2-devel libxslt-devel tar nodejs \
+	glibc-all-langpacks
```

Comment:
You can get UTF-8 with setting `export LC_ALL=C.UTF-8` in the container, and save yourself 19 MB in the container.

Reply:
Ah, good to know.